ros_packages/motors/pib_motors/pib_motors/motor.py

```diff
@@ -5,30 +5,43 @@
 
 class Motor:
 
-	def __init__(self, name: str, bricklet_pins: list[BrickletPin], invert: bool):
+	def __init__(self, name: str, bricklet_pins: list[BrickletPin], invert: bool, rotation_range_min: int, rotation_range_max: int):
```

Comment:
It is not necessary to add the rotation-range values as constructor parameters (this might give the impression, since their values are set in `apply_settings` anyway. I would simply set their respective values to the absolute possible min/max values in the constructor.

Reply:
Implemented as requested